### DISCUSSION TITLE
Funktionsprobleme durch überflüssiges Trennzeichen in api.config

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#64] [General]
Ein überflüssiger Strichpunkt (";") am Ende  des keys "ignore" in der Sektion "proxy" der api.config hat zu seltsamen Funktionsproblemen im WebGIS geführt. Das Ende des Strings sieht so aus: ...;server.vorarlberg.at;" /> Der überflüssige ";" ist sehr schwer zu entdecken.
U.a. wurden keine Höheninformationen mehr vom Voibos Höhenservice geliefert. In der webgis_exceptions.log wurden keine Fehler dokumentiert.
Mein Wunsch: Parsing Fehler in der Konfiguration sollten einen Niederschlag in einem Logfile finden.

**Comment by gis-eni** [comment]
Es ist halt sehr schwer Parsing Fehler auszugeben, wenn sie nicht als Fehler bekannt sind ;)
Hier wäre eher einmal zu hinterfragen, was genau das Problem war und wie es zustande kommt
